common/shlibs

```diff
@@ -3400,3 +3400,5 @@ libsnoresettings-qt5.so.0.7 snorenotify-0.7.0_1
 libTECkit.so.0 libteckit-2.5.8_1
 libTECkit_Compiler.so.0 libteckit-2.5.8_1
 libwf-config.so wf-config-0.1_1
+libretdec-libelf.so retdec-3.2_1
```

Comment:
please don't add these, they seem to be internal libraries

Reply:
What do you mean by 'internal libraries'?
`retdec-bin2llvmir`, `retdec-fileinfo` and `retdec-unpacker` link against `libretdec-libdwarf.so` and `libretdec-libelf.so`.

Reply:
They are in the same pacakge, so they don't need to be referenced in common/shlibs. A

Reply:
According the comment at the beginning of the shlibs file:
> Every shared library installed by a package must be listed here and mapped to a binary package.

Reply:
Also xbps-src complains with a warning when building the package if the libraries are not specified in shlibs. What are the rules then? It does not seem very consistent.

Reply:
The rules are to only add an entry to common/shlibs when it is actually needed.

Reply:
Do I need to understand the rule as "if there are not a lot of chances an external binary will link against those libraries then they should _not_ be listed in shlibs" ? If yes, then shlibs comment should definitely be fixed.